static-assets/components/cstudio-common/common-api.js

```diff
@@ -6579,7 +6580,7 @@ var nodeOpen = false,
       /**
        * dynamically add a javascript file
        */
-      addJavascript: function(script, cache, callback) {
```

Comment:
The change in the API is worrisome. Did you check the **entire** codebase for usages of addJavascript not using the original argument configuration? Did you searched for the "addJavascript" string alone or with its whole path?

Are cache and callback arts not used inside the function?

Reply:
I searched for the string `addJavascript` (alone) all over the code, and I found no usages with the optional arguments, also, in the code neither cache nor callback are being used